app/localization/src/DefaultVocabulary.js

```diff
@@ -30,6 +30,8 @@ const Translations = {
   'core.authentication.login': 'Login!',
   'core.authentication.logout': 'Logout',
 
+  'gallery.gallery_grid.no_images': 'No images available.',
```

Comment:
This gallery is still only for hotels so it should be prefixed as well (see `hotels.gallery.pagination`).